src/NzbDrone.Core/Parser/IsoLanguages.cs

```diff
@@ -26,6 +26,7 @@ public static class IsoLanguages
                                                                new IsoLanguage("fi", "fin", Language.Finnish),
                                                                new IsoLanguage("tr", "tur", Language.Turkish),
                                                                new IsoLanguage("pt", "por", Language.Portuguese),
+                                                               new IsoLanguage(new List<string> {"pt-br", "pt-BR"}, "pob", Language.Brazilian),
```

Comment:
This should just be the two digit code, br

Reply:
Two digit code br is for another language (Breton).
Brazilian Portuguese oficial iso code is pt-br (variant of pt (Portuguese of Portugal)